### DISCUSSION TITLE
HW 2 Question

### DISCUSSION THREAD

**Discussion by shawnveilleux** [cooperrc/advanced-dynamics#30] [General]
![image](https://github.com/cooperrc/advanced-dynamics/assets/143618436/2efebcd3-dd13-4c9e-9049-16b117e273a4)
For Homework 2, question 2, we are given that the links start horizontally with r_c = 2 i and both theta1 and theta 2 = 0 degrees. I just want to confirm, theta1 is the angle between the ground and link AB, and theta2 is the angle between the ground and link BC? Or is theta2 the angle between link AB and BC?

**Comment by shawnveilleux** [comment]
If theta2 is the angle between the ground and link BC, considering the links are the same length, wouldn't that mean that at all times theta1 = theta2?

**Reply by ewk19001** [reply]
That's what I was assuming as well.

**Reply by gwalley24** [reply]
I believe so

**Reply by travismalave** [reply]
Same here.
